doc/api/http.markdown

```diff
@@ -452,6 +452,11 @@ If `data` is specified, it is equivalent to calling
 If `callback` is specified, it will be called when the response stream
 is finished.
 
+### response.finished
+
+Boolean value that indicates whether the response has completed. Defaults
```

Comment:
I would say "starts as false", its not a default. Default makes it sound like it can start with some other value.